components/org.wso2.carbon.identity.claim.verification.endpoint/src/main/resources/api.identity.claim.verification.yaml

```diff
@@ -0,0 +1,300 @@
+swagger: '2.0'
+######################################################
+# Prolog
+######################################################
+info:
+  version: "1.0.0"
+  title: "WSO2 Identity Server Claim Verification Rest API"
+  description: |
+    This document specifies a **Claim Verification RESTfulAPI** for WSO2 **Identity Server** .
+    It is written with [swagger 2](http://swagger.io/).
+  contact:
+    name: "WSO2"
+    url: "http://wso2.com/products/identity-server/"
+    email: "architecture@wso2.com"
```

Comment:
should be wso2.org

Reply:
Fixed with https://github.com/wso2-extensions/identity-governance/pull/293